gremlin-driver/src/test/java/org/apache/tinkerpop/gremlin/driver/TestWSGremlinInitializer.java

```diff
@@ -136,5 +145,28 @@ private String returnSingleVertexResponse(final UUID requestID) throws Serializa
 
             return SERIALIZER.serializeResponseAsString(ResponseMessage.build(requestID).result(t).create());
         }
+
+        private String returnSimpleStringResponse(final UUID requestID, String message) throws SerializationException {
+            return SERIALIZER.serializeResponseAsString(ResponseMessage.build(requestID).result(message).create());
+        }
+
+        /**
+         * Captures and stores user_agent if included in header
+         */
+        @Override
+        public void userEventTriggered(final ChannelHandlerContext ctx, final Object evt) {
+            if(evt instanceof WebSocketServerProtocolHandler.HandshakeComplete) {
+                WebSocketServerProtocolHandler.HandshakeComplete handshake = (WebSocketServerProtocolHandler.HandshakeComplete) evt;
+                HttpHeaders requestHeaders = handshake.requestHeaders();
+                if(requestHeaders.contains(UserAgent.USER_AGENT_HEADER)) {
+                    userAgent = requestHeaders.get(UserAgent.USER_AGENT_HEADER);
+                    logger.info("UserAgent is " + userAgent);
```

Comment:
I don't think the logging is necessary since it doesn't seem to be useful for this test.